flexmeasures/api/v3_0/sensors.py

```diff
@@ -225,129 +225,128 @@ def trigger_schedule(  # noqa: C901
-                    ],
-                    "soc-minima" : [
-                        {
-                            "value": 15,
-                            "datetime" : "2015-06-02T14:00:00+00:00"
+                    {
+                        "start": "2015-06-02T10:00:00+00:00",
+                        "flex-model": {
+                            "soc-at-start": 12.1,
+                            "soc-unit": "kWh"
+                        }
+                    }
+
+                **Example request B**
+
+                This message triggers a 24-hour schedule for a storage asset, starting at 10.00am,
+                at which the state of charge (soc) is 12.1 kWh, with a target state of charge of 25 kWh at 4.00pm.
+                The global minimum and maximum soc are set to 10 and 25 kWh, respectively.
+                To guarantee a minimum SOC in the period prior to 4.00pm, local minima constraints are imposed (via soc-minima)
+                at 2.00pm and 3.00pm, for 15kWh and 20kWh, respectively.
+                Roundtrip efficiency for use in scheduling is set to 98%.
+                Storage efficiency is set to 99.99%, denoting the state of charge left after each time step equal to the sensor's resolution.
+                Aggregate consumption (of all devices within this EMS) should be priced by sensor 9,
+                and aggregate production should be priced by sensor 10,
+                where the aggregate power flow in the EMS is described by the sum over sensors 13, 14 and 15
+                (plus the flexible sensor being optimized, of course).
+                Note that, if forecasts for sensors 13, 14 and 15 are not available, a schedule cannot be computed.
+
+                .. code-block:: json
+
+                    {
+                        "start": "2015-06-02T10:00:00+00:00",
+                        "duration": "PT24H",
+                        "flex-model": {
+                            "soc-at-start": 12.1,
+                            "soc-unit": "kWh",
+                            "soc-targets": [
+                                {
+                                    "value": 25,
+                                    "datetime": "2015-06-02T16:00:00+00:00"
+                                },
+                            ],
+                            "soc-minima" : [
+                                {
+                                    "value": 15,
+                                    "datetime" : "2015-06-02T14:00:00+00:00"
+                                },
+                                {
+                                    "value": 20,
+                                    "datetime" : "2015-06-02T15:00:00+00:00"
+                                }
+                            ],
+                            "soc-min": 10,
+                            "soc-max": 25,
+                            "roundtrip-efficiency": 0.98,
+        ç                   "storage-efficiency": 0.9999,
```

Comment:
Does the new indentation improve anything? There is also a strange character here: `ç`.

Reply:
Thanks for noticing!  I don't know how the indentation got changed... Now it's fixed